lib/ansible/runner/connection_plugins/zone.py

```diff
@@ -68,6 +68,7 @@ def __init__(self, runner, host, port, *args, **kwargs):
         self.runner = runner
         self.host = host
         self.has_pipelining = False
+        self.SUPPORTED=['sudo', 'su', 'pbrun', 'pfexec']
```

Comment:
Since this module do not implement itself any of these method, shouldn't this use the constants.BECOME_METHODS ? ( since that's basically what would be implemented by ansible, so this will automatically make a new method usable by this module )